### PR TITLE
Add action to run single maintenance-job for debug purpose

### PR DESCRIPTION
**Pull request by Ben-Ho**

### CODE REVIEW

**Comment by nsams** [comment]
add exit at the end of the action, else you get an error

**Comment by fraxachun** [comment]
Add showJobsAction which echoes the all jobs, that way you can remove the filter parameter and instead use a job parameter

**Comment by Ben-Ho** [comment]
both solved :-)

File: Kwf/Controller/Action/Cli/Web/MaintenanceJobsController.php [hunk 1]
```diff
@@ -68,4 +68,27 @@ public function runDailyAction()
...
+        }
+        exit;
+    }
+
+    public function runJobAction()
+    {
+        $debug = $this->_getParam('debug');
+        $jobName = $this->_getParam('job');
+        foreach (Kwf_Util_Maintenance_Dispatcher::getAllMaintenanceJobs() as $job) {
+            if (strpos(get_class($job), $jobName) === false) continue;
```
Comment (nsams) [inline Kwf/Controller/Action/Cli/Web/MaintenanceJobsController.php]: wenn der parameter job heißt sollte er exakt passen, oder?

**Pull request merged**